portfolio/src/main/webapp/css/main.css

```diff
@@ -26,31 +31,40 @@
 
 /* header style */
 .header{
-  background-image: url('../images/header.jpg');
+  background-image: url('../images/backdrop.JPG');
```

Comment:
Nit: should be in alphabetical order

Reply:
I see this is an issue throughout, and looks like it was in the starter code, so I'll just include the style convention for future reference.

Alphabetize declarations.

Put declarations in alphabetical order in order to achieve consistent code in a way that is easy to remember and maintain.

Ignore vendor-specific prefixes for sorting purposes. However, multiple vendor-specific prefixes for a certain CSS property should be kept sorted (e.g. -moz prefix comes before -webkit).

ex:
background: fuchsia;
border: 1px solid;
-moz-border-radius: 4px;
-webkit-border-radius: 4px;
border-radius: 4px;
color: black;
text-align: center;
text-indent: 2em;